rabbitmq/datadog_checks/rabbitmq/data/conf.yaml.example

```diff
@@ -38,6 +38,12 @@ instances:
     #
     # tag_families: false
 
+    ## @param disable_node_metrics - boolean - optional - default: false
+    ## Use the disable_node_metrics paramater if you'd like to collect all
+    ## metrics except the ones pertaining to nodes
+    #
+    # disable_node_metrics: false
```

Comment:
In general _positive_ statements are easier to understand. I think is better to rename to `collect_node_metrics` and default to `true`.

Reply:
That makes sense- updated!